internal/rollup/uglify.config.json

```diff
@@ -1,7 +1,6 @@
 {
   "compress": {
     "pure_getters": true,
-    "pure_funcs": ["Object.defineProperty"],
```

Comment:
👍

Reply:
yeah this broke the code for new Angular renderer. I'll have to add a way for Angular (and others) to set their own uglify settings...